nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -245,15 +245,22 @@ in
     };
 
     acmeCA = mkOption {
-      default = "https://acme-v02.api.letsencrypt.org/directory";
+      default = null;
       example = "https://acme-staging-v02.api.letsencrypt.org/directory";
       type = with types; nullOr str;
       description = lib.mdDoc ''
+        ::: {.note}
+        Sets the [`acme_ca` option](https://caddyserver.com/docs/caddyfile/options#acme-ca)
+        in the global options block of the resulting Caddyfile.
+        :::
+
         The URL to the ACME CA's directory. It is strongly recommended to set
-        this to Let's Encrypt's staging endpoint for testing or development.
+        this to Let's Encrypt's [staging endpoint](https://letsencrypt.org/docs/staging-environment/)
+        for testing or development.
 
-        Set it to `null` if you want to write a more
-        fine-grained configuration manually.
+        Special value `null` should be prefered for production setups,
```

Comment:
```suggestion
        A value `null` should be prefered for production setups,
```

Reply:
I wanted to get the point across that this is a nixos-specific option value.
Not a Caddyfile value, considering the nixos module passes a lot of things directly to the resulting Caddyfile.

I could phrase it more clearly, sure.
But replacing "Special" with "a" feels like the opposite direction to me.